### PR TITLE
Fix build by adding /v4 to module path

### PR DESCRIPTION
**Pull request by yonas**
This PR will close:

- #252
- #208

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <p>
-    <a href="https://github.com/achannarasappa/ticker/releases"><img src="https://img.shields.io/github/v/release/achannarasappa/ticker" alt="Latest Release"></a>
-    <a href="https://github.com/achannarasappa/ticker/actions"><img src="https://github.com/achannarasappa/ticker/workflows/test/badge.svg" alt="Build Status"></a>
+    <a href="https://github.com/achannarasappa/ticker/v4/releases"><img src="https://img.shields.io/github/v/release/achannarasappa/ticker" alt="Latest Release"></a>
```
Comment (achannarasappa) [inline README.md:2]: URLs in the readme should remain without a `/v4/`

File: .goreleaser.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ brews:
     commit_author:
       name: achannarasappa
       email: git@ani.dev
-    homepage: "https://github.com/achannarasappa/ticker"
+    homepage: "https://github.com/achannarasappa/ticker/v4"
```
Comment (achannarasappa) [inline .goreleaser.yml:65]: This URL should remain without a `/v4/`

File: .goreleaser.yml [hunk 3]
```diff
@@ -115,7 +115,7 @@ nfpms:
     file_name_template: "{{ .ProjectName }}-{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}-{{ .Mips }}{{ end }}"
     builds:
       - build-linux-64
-    homepage: https://github.com/achannarasappa/ticker
+    homepage: https://github.com/achannarasappa/ticker/v4
```
Comment (achannarasappa) [inline .goreleaser.yml:118]: This URL should remain without a `/v4/`

File: .goreleaser.yml [hunk 4]
```diff
@@ -15,7 +15,7 @@ builds:
       post:
         - upx "{{ .Path }}"
     ldflags:
-      - -s -w -X 'github.com/achannarasappa/ticker/cmd.Version={{.Version}}'
+      - -s -w -X 'github.com/achannarasappa/ticker/v4/cmd.Version={{.Version}}'
```
Comment (achannarasappa) [inline .goreleaser.yml:18]: Pretty sure these should remain with a `/v4/` but not certain. Have you run a test build to confirm this should be here?

**Review by achannarasappa — CHANGES_REQUESTED** [review]
Thanks for the pull request! I've added a few comments

Also please rebase with master. I've fixed the linting and test errors that were present when this branch was originally created so you should be able to get clearer automated feedback on the PR.

**Comment by clausecker** [comment]
Any progress on this one?

**Comment by achannarasappa** [comment]
I think there might be some other issue causing problem here with `go install` but I've updated the module path.

**Pull request closed without merging**